components/apimgt/org.wso2.carbon.apimgt.keymgt/src/test/java/org/wso2/carbon/apimgt/keymgt/issuers/RoleBasedScopesIssuerTestCase.java

```diff
@@ -66,4 +83,489 @@ public void testGetAllowedScopesWhenAuthorizedScopesEmpty() throws Exception {
         Assert.assertEquals("default", authorizedScopes.get(0));
     }
 
+    @Test
+    public void testGetScopes() throws Exception {
+
+        ApiMgtDAO apiMgtDAO = Mockito.mock(ApiMgtDAO.class);
+        RealmService realmService = Mockito.mock(RealmService.class);
+        CacheManager cacheManager = Mockito.mock(CacheManager.class);
+        TenantManager tenantManager = Mockito.mock(TenantManager.class);
+        Cache cache = Mockito.mock(Cache.class);
+        DefaultRealm defaultRealm = Mockito.mock(DefaultRealm.class);
+        AbstractUserStoreManager abstractUserStoreManager = Mockito.mock(AbstractUserStoreManager.class);
```

Comment:
Shall we move common mocks to a @ Before method

Reply:
Fixed in 121cc9e994e389d43cc4b11ccabf7bddeadebc91.